app/controllers/spotlight/appearances_controller.rb

```diff
@@ -7,6 +7,11 @@ class AppearancesController < Spotlight::ApplicationController
 
     def update
       if @exhibit.update(exhibit_params)
+        # Update masthead attributes, only after we have saved the masthead_id
+        update_masthead
```

Comment:
Is it possible to push this down into the model? Or, at least, leave a comment about why it isn't?